src/sparse_int_set.jl

```diff
@@ -0,0 +1,252 @@
+import Base: @propagate_inbounds, zip
```

Comment:
imports should be at to of the main DataStructures.jl file.
and ratehr than `import` use `using`, and then to overload `zip` define the function as `Base.zip(`
```
import Base: @propagate_inbounds, zip
```

Reply:
robin_dict.jl and int_set.jl also import a lot of stuff there, they will also have to be cleaned up.

Reply:
yeah, we do still need to do more housekeeping on the repo.
But lets not make it worse